tests/nightly/model_backwards_compatibility_check/common.py

```diff
@@ -29,6 +29,13 @@
 import re
 from mxnet.test_utils import assert_almost_equal
 
+try:
```

Comment:
`cmp = lambda(x, y): (x > y) - (x < y)` could also work with a bit less code

Reply:
1. It is a Python 3 syntax error
2. With the syntax fixed it generates a PEP8 violation in flake8 https://stackoverflow.com/questions/25010167/e731-do-not-assign-a-lambda-expression-use-a-def

Reply:
Ok, I see. Thanks!